specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2018-11-20/guestconfiguration.json

```diff
@@ -802,16 +802,28 @@
           "type": "string",
           "description": "Version of the guest configuration."
         },
-        "contentUri": {
-          "readOnly": true,
+        "contentUri": {          
           "type": "string",
           "description": "Uri of the storage where guest configuration package is uploaded."
         },
-        "contentHash": {
-          "readOnly": true,
+        "contentHash": {          
           "type": "string",
           "description": "Combined hash of the guest configuration package and configuration parameters."
         },
+		"assignmentType": {
+			"type": "string",
+			"description": "Specifies the assignment type and execution of the configuration. Possible values are Audit, DeployAndAutoCorrect, ApplyAndAutoCorrect and ApplyAndMonitor.",
```

Comment:
BTW, now the assignmentType is writeable property, right? Seems I didn't see the attribiute "readOnly: false".

Reply:
by default its false.